src/DependenciesPackager/CrossgenContext.cs

```diff
@@ -192,22 +365,25 @@ private DirectoryInfo CreateSubDirectory(string cacheBasePath, PackageEntry pack
                 package.Library.Identity.Version.ToNormalizedString().ToLowerInvariant(),
                 Path.GetDirectoryName(asset.RelativePath));
 
-            _logger.LogInformation($"Creating sub directory on {subDirectoryPath}.");
-
             return Directory.CreateDirectory(subDirectoryPath);
         }
 
-        private string GetTempStagingFolder(string parent)
+        public void CreateResponseFile()
         {
-            var publishFolder = _context.GetStagingFolderPath(parent);
-
-            if (!Directory.Exists(publishFolder))
-            {
-                _logger.LogInformation($"Creating directory {publishFolder}.");
-                Directory.CreateDirectory(publishFolder);
-            }
+            _responseFilePath = Path.Combine(_destinationFolder, "aspnet-crossgen.rsp");
+            var builder = new StringBuilder();
+            builder.AppendLine($"/Platform_Assemblies_Paths \"{string.Join(";", _referenceAssemblyPaths)}\"");
```

Comment:
You mean because of the string.Join()?  It's the first time i'm writing a response file, It probably works with all in new lines although I see `/flag parameters` as the argument.  I say we keep it this way, it doesn't make a big difference to have 3 lines vs 6 lines on the file